### DISCUSSION TITLE
Upgrading Node the the latest lts version

### DISCUSSION THREAD

**Discussion by mcharnet** [vercel/community#2265] [Bug Report]
When selecting version 18 of Node in the Vercel configuration, we do not have the last lts version. Indeed, it seems to be 18.13 instead of 18.16. Is it possible to upgrade, or tell me how to choose a specific version ? Thanks.

**Comment by swarnava** [comment]
Hi there,

At this point, we have only Node 18.13 available in Build runtime. We are working with upstream provider to upgrade it to Node 18.15.0 in next few weeks. Unfortunately, we don't have any concrete ETA on this.

**Comment by swarnava** [comment]
We just released Node 18.15.0 in our build container.
